std_msgs/Header header

string path
string jsonData
